src/lib/serviceRoutines/deleteIndividualContextEntity.cpp

```diff
@@ -90,7 +93,10 @@ std::string deleteIndividualContextEntity
   }
 
   // 05. Cleanup and return result
+  TIME_STAT_RENDER_START();
```

Comment:
In mi opinion, readibility is similar ni both cases. Although `answer = response.render(ciP->outFormat, "", false, false)` is slighly more readable than `renderWithTimeStat(answer, response, render, (ciP->outFormat, "", false, false))` the TIME_STAT_RENDER_START() and TIME_STAT_RENDER_STOP() lines add readbility "noise".

Thus, being readibility more or less the same, I think the alternative of using a macro is safer. Otherwise, we can miss for example a  TIME_STAT_RENDER_STOP() when writting the code.

(BTW, I'm not sure that renderWithTimeStat is the best name... maybe you can find some better).